mautic/mautic
# Update filemanager

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | https://github.com/helios-ag/FMElfinderBundle
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8259
| BC breaks? | The old filemanager is removed.
| Deprecations? | The old filemanager is removed. 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Mautic's filemanager is EOL and the replacement doesn't look very active.

We could use something like https://github.com/helios-ag/FMElfinderBundle which uses Elfinder.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to the filemanager and try to use it. It should be working.

#### List deprecations along with the new alternative:
1. The old filemanager has been removed.
2.  All the related jQuery plugins needed for the old filemanager have aslo been removed.

#### List backwards compatibility breaks:
1. The old filemanager has been removed.
